cinema/views.py

```diff
@@ -0,0 +1,38 @@
+from rest_framework.decorators import api_view
+from rest_framework.generics import get_object_or_404
+from rest_framework.response import Response
+from rest_framework import status
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movie_list(request):
+    if request.method == 'GET':
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+    else:
+        serializer = MovieSerializer(data=request.data)
+        if serializer.is_valid():
+            serializer.save()
```

Comment:
use [raise_exception=True ](https://stackoverflow.com/questions/40348981/django-rest-framework-raise-exception-true)parameter to avoid returning Bad Request